examples/get-cert.py

```diff
@@ -16,18 +16,21 @@
     load_privx_api_lib_path()
     import privx_api
 
-
-PUB_KEY_PATH = "path to the public key"
-CERT_PREFIX = "ssh-rsa-cert-v01@openssh.com"
-
+# The parameters of the API call
+# NOTE: Make sure that you have the "Use with PrivX Agent" permisson in your role
 conf = {
-    "public_key": "public key",
-    "role_id": "a role UUID with which to request "
-    "the certificate (client must be a member)",
-    "hostid": "target host UUID",
-    "hostname": "target host hostname",
-    "username": "target username",
-    "service": "SSH",
+    # User's public key without padding (MANDATORY)
```

Comment:
mandatory in the POST payload but not here in the conf. Since main() cleans up the padding this requirement (here, in conf{}) is not necessary. To emphasise the need to normalise the public key, add a function (one-liner) to do the clean up instead of doing it in the main function:

```
# Normalise the public key to the format without padding
def no_padding(pubkey: string) -> string:
     return ...
```

main():
    conf["public_key"] = no_padding(conf["public_key"])